CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ and this project adheres to
   supported snapshot version format. This change renders all previous
   Firecracker snapshots (up to Firecracker version v1.6.0) incompatible with the
   current Firecracker version.
+- [#4449](https://github.com/firecracker-microvm/firecracker/pull/4446): Added
```

Comment:
a mismatch between the text and the link (4449 vs 4446)

Reply:
ah, good catch, fixed!